cypress/support/config/settings.js

```diff
@@ -4299,7 +4297,7 @@ module.exports = () => ({
           local: {
             paths: [
               // '/swahili/media-23268999', // CPS MAP with live stream
-              '/swahili/michezo/2016/07/160713_tc2_testmap2', // TC2 MAP with audio clip
+              '/swahili/michezo/2016/07/160712_tc2_testmap1', // TC2 MAP with video clip
```

Comment:
This asset doesn't exist locally - do we also need to add it?

Reply:
(Might just be easier to remove instead)

Reply:
Oops, will add